inspirehep/modules/workflows/workflows/article.py

```diff
@@ -219,19 +213,28 @@
 
 CHECK_IF_SUBMISSION_AND_ASK_FOR_APPROVAL = [
     IF_ELSE(
-        is_record_relevant,
-        [halt_record(
-            action="hep_approval",
-            message="Submission halted for curator approval.",
-        )],
+        is_record_merged,
```

Comment:
@kaplun I changed this part of the workflow in order to:

-  Do not block the workflow if we are merging 2 articles and there are no conflicts
-  Block the workflow if we are merging 2 articles and there are conflicts
-  Maintain the same behaviour for new articles

Let me know if it makes sense for you